10 lab

KachalkinDmitriy opened the pull request:
тимлид не может прислать, поэтому присылаю я. В архиве кидаю так как без него файл не загружается.

KachalkinDmitriy committed:
10 lab

тимлид не может прислать, поэтому присылаю я. В архиве кидаю так как без него файл не загружается.

NemirovichMark commented:
Здравствуйте, Дмитрий.
Жаль. Ладно.
Поправить UML:
Состояний. У вас в main первым создается массив BoardGame. Значит, в диаграмме состояний он должен идти перед незаполненным массивом GameCatalog. 
Процессов. Пустых концов не должно быть
![image](https://github.com/NemirovichMark/BBI-23-2/assets/112826355/8db378e9-245a-43ba-9781-f913377a1bca)
Классов. У вас не композиция, а агрегация.
![image](https://github.com/NemirovichMark/BBI-23-2/assets/112826355/88dbdb9f-392c-42ab-afc8-84feebd9ee59)

Код принял.
Доп. компоненты не будете делать, правильно понял?

На доработку.

KachalkinDmitriy commented:
я не уверен что правильно понял ваши исправления, вы написали что в main первым создается массив BoardGame. Значит, в диаграмме состояний он должен идти перед незаполненным массивом GameCatalog. Это получается мне нужно показывать как заполнялся массив BoardGame, или просто написать перед GameCatalog что сначала создался массив BoardGame?
![image](https://github.com/NemirovichMark/BBI-23-2/assets/159521740/de40dcbe-e8b8-4632-a43e-18876622a79e)

NemirovichMark commented:
Здравствуйте, Дмитрий.
Да, так:
"Это получается мне нужно показывать как заполнялся массив BoardGame", как вы сделали.

KachalkinDmitriy committed:
9 lab

KachalkinDmitriy commented:
здравствуйте, загрузил 9 лабу

NemirovichMark commented:
Здравствуйте, принял 9 лабу

KachalkinDmitriy committed:
Диаграммы для 10 лабы

В диаграмме классов использовал только агрегацию, реализацию и наследование. В диаграмме состояний массив BoardGame в нашем коде заполняется циклом и никаких других действий мы с ним не проделываем. Диаграмма процессов больше не имеет пустых концов.

NemirovichMark commented:
Здравствуйте, Дмитрий. Принял 10ю, пусть девушки тоже загружают и отправляют.
